README.md

```diff
@@ -246,14 +274,50 @@ mgit exec 'echo `pwd`'
 # /home/mgit/packages/organization/repository-2
 ```
 
-### save-hashes
+### commit (alias: `ci`)
+
+For every repository that contains changes which can be committed, makes a commit with these files. 
+You need to specify the message for the commit.
+
+Example:
+
+```bash
+mgit commit -- --message 'Introduce PULL_REQUEST_TEMPLATE.md.'
+
+# Executes `git commit --message 'Introduce PULL_REQUEST_TEMPLATE.md.'` command on each repository.
+# Commit will be made in repositories that "git status" returns a list if changed files (these files must be tracked by Git).
+```
+
+### merge
+
+Requires a second argument which is a branch name that will be merged to current one. You can also specify the message
+which will be added to the default git-merge message.
+
+Repositories which do not have specified branch will be ignored.
+
+Example:
+
+```bash
+# Assumptions: we are on "master" branch and "develop" branch exists.
```

Comment:
Here you used `"` and below backticks around branch names. Since this is preformatted anyway, I'd go with `"`.